.github/ISSUE_TEMPLATE/writing_review.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Writing Review
+about: Request feedback on your writing
+title: '[Writing] '
+labels: 'Status:1-New, Type:Feature'
+assignees: ''
+
+---
+# Review Details
+<!-- Review guide for requesting a review: https://arfc.github.io/manual/guides/katy/revreq. -->
+
+**Submission Deadline**: <!-- Indicate if self-imposed or external. -->
+
+**Intended Venue**:
+
+**Link to pdf**:
+
+**Options for receiving feedback**: <!-- Default to however the reviewer is willing to provide it. -->
+
+**Anticipated response time to feedback**: <!-- How long will it take for you to respond to feedback? -->
+
+**Have you applied the [Writing Checklist](https://arfc.github.io/manual/guides/writing/checklist/)?**:
+
+
+## What is the pre-discussed timeline?
+
+
+
+## What review has this gone through so far?
+
+
+
+## Additional context.
+
+<!-- Is there something you would like the reviewer to focus on specifically? -->
+
+# How can this issue be closed?
+
+<!-- Identify people related to this discussion, appropriate tags, and projects in the group org. -->
```

Comment:
I think this may overlap with `Additional Context`.

Reply:
I can see that! I was thinking `Additional Context` would be for things like: say the version you're asking for review on is a little shaky in the abstract in particular and you want the reviewer to give a little extra attention to that particular part. 

In comparison, `How can this issue be closed?` is standard across all of our issue templates, and it's a shop-keeping thing for issues themselves. In this section, you could indicate if you want to make a new issue for each round of review or just keep the same issue open. 

Maybe there's a better section, sub-section header, or comment that could make this distinction clearer? Or they could be rolled together, but in my mind one is about the document and the other is about the issue.

Reply:
Ah, I see what you mean. Maybe we could adjust the comment to make that more obvious, something like `Identify people related to this discussion, appropriate tags, projects in the group org, and other meta topics.`